src/generators/clientFileGenerator.ts

```diff
@@ -518,6 +520,38 @@ function getTrack1DefaultContent(
   };
   super(optionsWithDefaults);
   `;
+
+  if (azureArm && !allowInsecureConnection) {
+    return (
+      defaultContent +
+      `
+      if (options?.pipeline && options.pipeline.getOrderedPolicies().length > 0) {
+        const pipelinePolicies: coreRestPipeline.PipelinePolicy[] = options.pipeline.getOrderedPolicies();
```

Comment:
```suggestion
        const pipelinePolicies = options.pipeline.getOrderedPolicies();
```
This might make it easier to read.